src/interfaces/coral_web/src/components/Citations/CitationDocument.tsx

```diff
@@ -31,7 +31,7 @@ export const CitationDocument: React.FC<Props> = (props) => {
       })}
     >
       <CitationDocumentHeader
-        toolId={document.tool_name?.toLowerCase() ?? undefined}
+        toolId={document.tool_name ?? undefined}
```

Comment:
no longer need this because I updated `constants.ts` tool ids to match what the backend gives us